playground-common/src/Playground/Types.hs

```diff
@@ -113,6 +113,7 @@ type Expression = ContractCall JSON.Value
 data Simulation =
     Simulation
         { simulationName    :: String
+        , simulationIndex   :: Int
```

Comment:
I'm a little bothered by this name. Index is a loaded term, and `simulationIndex` implies to me that #3 is the third simulation, which isn't always true. (Create 3, delete the first 2, and the simulation with index 3 is actually at position 1.)

Is `simulationSequenceNumber` better? `simulationCreationId` or `simulationCreationTime`? 

(For the record I think it's fine for a `...Time` field to be an `Int` in a _typed_ language. I admit it would be confusing in an untyped one.)

I think I like `simulationSequenceNumber` best, but I throw it open to the floor...

Reply:
I take your point. But how about just `simulationNumber`? And a comment explaining what it's for (i.e. fixing the `simulationName` of new simulations).